PoseEstimation-TFLiteSwift/PoseEstimator.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  PoseEstimator.swift
+//  PoseEstimation-TFLiteSwift
+//
+//  Created by Doyoung Gwak on 2020/03/14.
+//  Copyright © 2020 Doyoung Gwak. All rights reserved.
+//
+
+import CoreVideo
+
+enum PoseEstimationResult {
```

Comment:
have you ever seen, https://developer.apple.com/documentation/swift/result ?
I think, you need to implement no fail case.

Reply:
Cool. Thanks for your advice.